lib/generator/exercise_case/case_helpers.rb

```diff
@@ -26,6 +26,15 @@ def indent_heredoc(lines, delimiter, depth = 0, delimiter_method = nil)
         ].join("\n")
       end
 
+      # combine array of string elements into a single string
+      # as part of workload with optional separator
+      #
+      #  example usage:  on_single_line(["foo", "bar"])
+      #  example output: "foo\nbar"
+      def on_single_line(lines, separator="\n")
+        lines.join(separator).inspect
+      end
```

Comment:
Needs a test to catch if this behavior changes in a way that is not wanted.

Reply:
I've added a few tests to catch behavior with strings, and with mixed classes (string, integer, nil) in the latest version.